docs/site/HasMany-relation.md

```diff
@@ -0,0 +1,184 @@
+---
+lang: en
+title: 'Relations'
+keywords: LoopBack 4.0, LoopBack 4
+tags:
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/HasMany-relation.html
+summary:
+---
+
+## Overview
+
+A `hasMany` relation denotes a one-to-many connection of a model to another
+model through referential integrity. The referential integrity is enforced by a
+foreign key constraint on the target model which usually references a primary
+key on the source model. This relation indicates that each instance of the
+declaring or source model has zero or more instances of the target model.
+
+## Defining a HasMany Relation
+
+To add a `HasMany` relation to your LoopBack application, you need to perform
```

Comment:
it seems like the steps fit better in the overview, because this section is already about the first step. Just a suggestion

Reply:
Good catch! That is where it should be :-).